### ISSUE TITLE
MI MIX 2 Accidently pressed update ramdisk in my main ROM and restarted device. No way to get back into OS. Stuck in Fastboot Mode. 

### ISSUE THREAD

**Issue by phooooon** [chenxiaolong/DualBootPatcher#1381]
Heres what i did:
1. unlocked my phone
2. Installed twrp
3. Installed dual boot patcher
4. selected my main ROM (miui 10) 
5. pressed update ramdisk by mistake
6. restarted my phone not knowing what it was going to do

When i restarted it went into fastboot mode and i do not know what to do now, i have twrp if that helps

I HAVE TWRP
I tried to factory reset in twrp but it didnt work

**Comment by phooooon** [comment]
MI MIX 2 Accidently pressed update ramdisk in my main ROM and restarted device. No way to get back into OS. Stuck in Fastboot Mode.

**Comment by phooooon** [comment]
Sorry, FIXED

I'm so stupid. I downloaded a random rom and installed it using twrp...


it worked and now i have an OS

**Comment by klorinczi** [comment]
You just fixed the phone booting problem, not the problem source. 
Likely the problem source is, that latest version of DualBootPatcher causes bootloop. 

Unfortunately I also noticed that a flashed rom goes into bootloop after I patched with latest DualBootPatcher and rebooted. 

I had to flash an earlier boot.img to fix the bootloop.

**Comment by yyk419476391** [comment]
Same problem in Xiaomi Redmi 5 Plus (vince)
AospExtended Pie v6.2 and Magisk v18.1

I installed DBP normally and seted kernel.
Then I update ramdisk and reboot.
But it boots in to fastboot mode.

When I flash my Boot's backup, I can enter my system again.

**Comment by realkot** [comment]
Mi mix 2
Updating the ramdisk causes the botloop into fastboot mode.
Restore the kernel backup abd all ok after that. But dualboot isn't working.

**Comment by phooooon** [comment]
Yeah, i cant dualboot ANY rom. If i attempt to add a second rom, the app crashes on all versions
